client/packages/host/src/Admin/WebAppLogFileModal.tsx

```diff
@@ -20,27 +21,30 @@ export const LogDisplay = ({
   fileName: string;
   setLogContent: (content: string[]) => void;
 }) => {
-  const { data, isLoading } = useLog.document.logContentsByFileName(fileName);
+  const { mutateAsync, data, isLoading } =
+    useLog.document.logContentsByFileName();
```

Comment:
I'm really curious about this mutateAsync. 

I found I was struggling to get the hooks working with useEffect in the parent function, so I'd love to know more about this and how it works